test-case-reporting/types/test-case-reporting.d.ts

```diff
@@ -22,6 +24,8 @@ declare module 'test-case-reporting' {
 
   export type TestStatus = 'PASS' | 'FAIL' | 'SKIP' | 'ERROR';
 
+  export type QueryFunction = (q: QueryBuilder) => QueryBuilder;
```

Comment:
This could probably be defined in the file that uses it, since it's only relevant to that module

Reply:
Done.